spec/request/swagger_spec.rb

```diff
@@ -26,7 +26,7 @@
   subject { Apivore::SwaggerChecker.instance_for('/v0/apidocs.json') }
 
   let(:rubysaml_settings) { build(:rubysaml_settings) }
-  let(:mhv_user) { build(:user, :mhv) }
+  let(:mhv_user) { build(:user, :mhv, middle_name: 'Bob') }
```

Comment:
I'm not completely clear why this needed to be added in this PR -- seems unexpected so therefore 🤔 

Did something change in the factory?

Reply:
Answered my own question -- there's a new field (middle name) in the swagger docs above that I missed. Carry on!

@annaswims Would it make sense to put middle name in the factory itself?

Feels like putting it in the `let` here makes it something I should care about looking at for the spec here (why is middle_name set? did I miss something? why is it the special attribute compared to everything else). 

Of course adding it to the factory might have other ramifications?

Reply:
Honestly, I'd consider it a bit of a cheat. I documented the `middle_name` attribute, but the test was failing because `middle_name` was `null` when I said it was a string.  Perhaps we want to accept `null` as an acceptable value in these tests when the docs say it's a string.

I was afraid if I changed the factories to have middle names that I'd fail all sorts of other tests.